fetch.bs

```diff
@@ -1179,36 +1179,42 @@ or "<code>worker</code>".
  <li><p>If <var>body</var> is null, then <a>queue a fetch task</a> on <var>request</var> to
  <a>process request end-of-body</a> for <var>request</var> and abort these steps.
 
+ <li><p>Let <var>read</var> be the result of <a lt="read a chunk" for=ReadableStream>reading a
+ chunk</a> from <var>body</var>'s <a for=body>stream</a>.
+
  <li>
-  <p>Let <var>read</var> be the result of <a lt="read a chunk" for=ReadableStream>reading a
-  chunk</a> from <var>body</var>'s <a for=body>stream</a>.
+  <p>Run these steps repeatedly, <a>in parallel</a>
```

Comment:
I'd make this "In parallel, while true:" to make it clear you go in parallel once and then keep iterating.